tests/unit/update-manager-test.js

```diff
@@ -198,4 +197,53 @@ describe('UpdateManager', () => {
 
   })
 
+  describe('editable', () => {
+    const self = {err: ''}
+    let updater, docuri, rterm, options, userCallback, loadStub
+    var loadStatus = 200
+
+    beforeEach(() => {
+      options = {}
+      userCallback = () => {}
+
+      updater = new UpdateManager()
+      // updater.store.fetcher.webOperation = sinon.stub().resolves({ ok: true, status: 200, statusText: "Dummy stub 1"})
+      // updater.store.fetcher.load = sinon.stub().resolves({ ok: true, status: 200, statusText: "Dummy stub 2"})
+
+    })
+
+    it('Should detect a document is editable from metadata', () => {
+      loadMeta(updater.store)
+      expect(updater.editable(doc1)).to.equal('SPARQL')
+    })
+
+    it('Should not detect a document is editable from metadata after flush', () => {
+      loadMeta(updater.store)
+      updater.flagAuthorizationMetadata()
+      expect(updater.editable(doc1)).to.equal(undefined)
+    })
+
+
+    it('Async version should detect a document is editable from metadata', () => {
+      loadMeta(updater.store)
+      updater.checkEditable(doc1).then(
+        res => expect(result).to.equal('SPARQL')
+      )
+      expect(updater.editable(doc1)).to.equal('SPARQL')
+    })
+
+    it('Async version should not detect a document is editable from metadata after flush', () => {
```

Comment:
I initially thought the logic is the other way around: if the triples are flagged as outdated it means the doc is editable.

Reply:
No,  many docs out there on many web servers are not editable.  So the default has to be not editable.  We have to find evidence from interaction with the document that it is in fact editable by the user in their current state.